### DISCUSSION TITLE
Toggleable() is still showing even though HiddenOn() is already applied.

### DISCUSSION THREAD

**Discussion by sasori13L** [filamentphp/filament#12271] [Help]
### Package

Table builder

### Package Version

v3.2.0

### How can we help you?

Toggleable() is still showing in relationmanager page
![image](https://github.com/filamentphp/filament/assets/16425810/131fa681-48f4-4d8b-a24b-988bcc109fbc)
![image](https://github.com/filamentphp/filament/assets/16425810/a9f3aeb7-9a10-4c71-9dce-ddb7b477ff0d)

**Comment by AllTheUserNamesAreGone** [comment]
I normally use 

``` ->toggleable(isToggledHiddenByDefault: true) ```

**Reply by sasori13L** [reply]
Correct but what I did was, I used hiddenOn() method so that the specific column will not show up whenever I use it in a relation-manager. though yes it does not show up in my relation-manager but in my show toggle columns it is still in there. though it does not do anything even if I check the toggle.

**Comment by AllTheUserNamesAreGone** [comment]
Replace what you have with this. It works for me and I can toggle.

```->toggleable(fn () => get_class() !== 'ProductResource\RelationManagers\ProductTransactionsRelationManger' ) ```

**Reply by sasori13L** [reply]
tried it but still doesn't work since the table is being accessed from a relation manager. The get_class() function only returns the owner resource url and not the one used in relation manager.

**Reply by AllTheUserNamesAreGone** [reply]
Oh I see, so the table is not defined in the ProductTransactionsRelationManger itself but is being used by it from elsewhere. 

I used debug_backtrace to see if the class came up in the call stack but it didn't for me, so I'm out of ideas.  

Here's the code I used to check if it's of interest.

```
        $needle = 'ProductResource\RelationManagers\ProductTransactionsRelationManger';
        $haystack = debug_backtrace();

        for ( $i=1; $i<count( $haystack ); $i++ ) {
            if(isset($haystack[$i]['class']))
               if ($needle === $haystack[$i]['class']) dd($needle);
        }
```

**Reply by sasori13L** [reply]
Thanks for the reply. I think this is a bug because it should not show up since it's hiddenOn() method is enabled.
